docs/guides/training.md

```diff
@@ -0,0 +1,21 @@
+# Training
```

Comment:
A link probably needs adding to this page from the main content page `docs/README.md`

Reply:
Done